sorting/quick_sort.py

```diff
@@ -0,0 +1,120 @@
+# Python3 program to perform basic timSort
```

Comment:
I think file name should be tim_sort.py

Reply:
There is already another file named the same